packages/frama-c/frama-c.20170501/opam

```diff
@@ -0,0 +1,72 @@
+opam-version: "1.2"
+name: "frama-c"
+version: "20170501"
+maintainer: "francois.bobot@cea.fr"
+authors: [
+  "Michele Alberti"
+  "Gergö Barany"
+  "Patrick Baudin"
+  "François Bobot"
+  "Richard Bonichon"
+  "David Bühler"
+  "Loïc Correnson"
+  "Julien Crétin"
+  "Pascal Cuoq"
+  "Zaynah Dargaye"
+  "Jean-Christophe Filliâtre"
+  "Philippe Herrmann"
+  "Florent Kirchner"
+  "Tristan Le Gall"
+  "Jean-Christophe Léchenet"
+  "Matthieu Lemerre"
+  "David Maison"
+  "Claude Marché"
+  "André Maroneze"
+  "Benjamin Monate"
+  "Yannick Moy"
+  "Anne Pacalet"
+  "Valentin Perrelle"
+  "Guillaume Petiot"
+  "Virgile Prevosto"
+  "Armand Puccetti"
+  "Muriel Roger"
+  "Julien Signoles"
+  "Kostyantyn Vorobyov"
+  "Boris Yakobowski"
+]
+homepage: "http://frama-c.com/"
+license: "GNU Lesser General Public License version 2.1"
+dev-repo: "https://github.com/Frama-C/Frama-C-snapshot.git"
+doc: ["http://frama-c.com/download/user-manual-Phosphorus-20170501.pdf"]
+bug-reports: "https://bts.frama-c.com/"
+tags: [
+  "deductive"
+  "program verification"
+  "formal specification"
+  "automated theorem prover"
+  "interactive theorem prover"
+  "C"
+  "plugins"
+  "abstract interpretation"
+  "slicing"
+  "weakest precondition"
+  "ACSL"
+  "dataflow analysis"
+]
+
+depends: [
+  "frama-c-base" { = "20170501" }
+  "lablgtk" { >= "2.18.2" } #for ocaml >= 4.02.1
+  "conf-gtksourceview"
+  "conf-gnomecanvas"
+  "alt-ergo"
+]
+
+depopts: [
+  "altgr-ergo" { build }
+]
+
+messages: [
+   "Alt-Ergo Graphical Interface can be used by the WP plug-in" { !altgr-ergo:installed }
+]
+available: [ ocaml-version >= "4.02.3" ]
```

Comment:
Thank you for this major release! It needs an `depends` field on `conf-autoconf` so that the external `autoconf` tool can be installed. This error is happening in CI:

```
#=== ERROR while installing frama-c-base.20170501 =============================#
# opam-version         1.2.2 (58ef3b8213100953848d362f7120a30356d7f77d)
# os                   linux
# command              make -j2
# path                 /home/opam/.opam/4.03.0/build/frama-c-base.20170501
# compiler             4.03.0
# exit-code            2
# env-file             /home/opam/.opam/4.03.0/build/frama-c-base.20170501/frama-c-base-29-2913ab.env
# stdout-file          /home/opam/.opam/4.03.0/build/frama-c-base.20170501/frama-c-base-29-2913ab.out
# stderr-file          /home/opam/.opam/4.03.0/build/frama-c-base.20170501/frama-c-base-29-2913ab.err
### stdout ###
# [...]
# Generating   src/plugins/inout/.Makefile.plugin.generated
# Generating   src/plugins/postdominators/.Makefile.plugin.generated
# Generating   src/plugins/constant_propagation/.Makefile.plugin.generated
# Generating   src/plugins/users/.Makefile.plugin.generated
# Generating   src/plugins/from/.Makefile.plugin.generated
# Generating   src/plugins/rte/.Makefile.plugin.generated
# Generating   src/plugins/occurrence/.Makefile.plugin.generated
# Generating   src/plugins/value/.Makefile.plugin.generated
# Generating   src/plugins/callgraph/.Makefile.plugin.generated
# Generating   src/plugins/metrics/.Makefile.plugin.generated
### stderr ###
# make: autoconf: Command not found
# make: *** No rule to make target 'configure', needed by 'config.status'.  Stop.
# make: *** Waiting for unfinished jobs....

```